diff -up opencv-2.4.7/cmake/OpenCVGenPkgconfig.cmake.pkgcmake opencv-2.4.7/cmake/OpenCVGenPkgconfig.cmake
--- opencv-2.4.7/cmake/OpenCVGenPkgconfig.cmake.pkgcmake	2013-11-06 15:26:39.000000000 +0100
+++ opencv-2.4.7/cmake/OpenCVGenPkgconfig.cmake	2013-11-13 23:19:49.414798348 +0100
@@ -50,8 +50,10 @@ foreach(CVLib ${OpenCV_LIB_COMPONENTS})
   else()
     set(installDir "${OPENCV_LIB_INSTALL_PATH}")
   endif()
-
-  set(OpenCV_LIB_COMPONENTS_ "${OpenCV_LIB_COMPONENTS_} \${exec_prefix}/${installDir}/${libname}")
+  string(REPLACE "libopencv" "-lopencv" libname "${libname}")
+  string(REPLACE ".so"    "" libname "${libname}")
+  string(REPLACE ".dylib" "" libname "${libname}")
+  set(OpenCV_LIB_COMPONENTS_ "${OpenCV_LIB_COMPONENTS_} ${libname}")
 endforeach()
 
 # add extra dependencies required for OpenCV
