diff -up opencv-3.4.1/samples/opencl/CMakeLists.txt.omv~ opencv-3.4.1/samples/opencl/CMakeLists.txt
--- opencv-3.4.1/samples/opencl/CMakeLists.txt.omv~	2018-06-11 20:51:10.642865138 +0200
+++ opencv-3.4.1/samples/opencl/CMakeLists.txt	2018-06-11 20:52:11.733868484 +0200
@@ -6,6 +6,11 @@ if(CMAKE_VERSION VERSION_LESS "3.1")
   return()
 endif()
 
+# FIXME for some reason, we get unresolved symbols on OpenCL bits even
+# though -lOpenCL is on the command line and in the right place...
+# Since this is only a sample, debugging this is not super high priority
+return()
+
 set(OPENCV_OPENCL_SAMPLES_REQUIRED_DEPS
   opencv_core
   opencv_imgproc
