--- opencv-3.4.0/cmake/OpenCVCompilerOptimizations.cmake.omv~	2018-01-12 01:41:41.500200915 +0100
+++ opencv-3.4.0/cmake/OpenCVCompilerOptimizations.cmake	2018-01-12 01:41:47.120213733 +0100
@@ -510,11 +510,7 @@ endmacro()
 macro(ocv_compiler_optimization_options_finalize)
   if(CMAKE_COMPILER_IS_GNUCXX AND (X86 OR X86_64))
     if(NOT APPLE AND CMAKE_SIZEOF_VOID_P EQUAL 4)
-      if(OPENCV_EXTRA_CXX_FLAGS MATCHES "-m(sse2|avx)")
-        add_extra_compiler_option(-mfpmath=sse) # !! important - be on the same wave with x64 compilers
-      else()
-        add_extra_compiler_option(-mfpmath=387)
-      endif()
+      add_extra_compiler_option(-mfpmath=sse) # !! important - be on the same wave with x64 compilers
     endif()
   endif()
 
