diff -uraN opencv-4.9.0/modules/dnn/CMakeLists.txt omv-opencv-4.9.0/modules/dnn/CMakeLists.txt
--- opencv-4.9.0/modules/dnn/CMakeLists.txt	2023-06-28 13:53:33.000000000 +0200
+++ omv-opencv-4.9.0/modules/dnn/CMakeLists.txt	2023-07-16 11:58:51.878820904 +0200
@@ -143,8 +143,10 @@
   #      COMMAND flatbuffers::flatc --cpp -o "${CMAKE_CURRENT_BINARY_DIR}" "${CMAKE_CURRENT_LIST_DIR}/src/tflite/schema.fbs")
 endif()
 
+add_definitions(-DPROTOBUF_USE_DLLS)
+
 list(APPEND include_dirs ${fw_inc})
-list(APPEND libs ${Protobuf_LIBRARIES})
+list(APPEND libs ${Protobuf_LIBRARIES}  absl_hash absl_log_internal_check_op absl_log_internal_message absl_raw_hash_set absl_raw_logging_internal absl_spinlock_wait absl_synchronization absl_log_internal_nullguard)
 if(NOT BUILD_PROTOBUF)
   list(APPEND include_dirs ${Protobuf_INCLUDE_DIRS})
 endif()

diff -uraN opencv-4.9.0/opencv_contrib-4.9.0/modules/ovis/CMakeLists.txt omv-opencv-4.9.0/opencv_contrib-4.9.0/modules/ovis/CMakeLists.txt
--- opencv-4.9.0/opencv_contrib-4.9.0/modules/ovis/CMakeLists.txt	2023-06-28 11:48:14.000000000 +0200
+++ omv-opencv-4.9.0/opencv_contrib-4.9.0/modules/ovis/CMakeLists.txt	2023-07-16 13:44:44.970722794 +0200
@@ -19,12 +19,14 @@
 include_directories(${OGRE_INCLUDE_DIRS})
 link_directories(${OGRE_LIBRARY_DIRS})
 
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../../../modules/objdetect/include)
+
 ocv_add_module(ovis opencv_core opencv_imgproc opencv_calib3d WRAP python)
 ocv_glob_module_sources()
 ocv_module_include_directories()
 ocv_create_module()
 
-ocv_add_samples(opencv_aruco)
+#ocv_add_samples(opencv_aruco)
 
 ocv_warnings_disable(CMAKE_CXX_FLAGS -Wunused-parameter)
 ocv_target_link_libraries(${the_module} ${OGRE_LIBRARIES})
diff -uraN opencv-4.9.0/opencv_contrib-4.9.0/modules/ovis/samples/aruco_ar_demo.cpp omv-opencv-4.9.0/opencv_contrib-4.9.0/modules/ovis/samples/aruco_ar_demo.cpp
--- opencv-4.9.0/opencv_contrib-4.9.0/modules/ovis/samples/aruco_ar_demo.cpp	2023-06-28 11:48:14.000000000 +0200
+++ omv-opencv-4.9.0/opencv_contrib-4.9.0/modules/ovis/samples/aruco_ar_demo.cpp	2023-07-16 13:45:47.378409705 +0200
@@ -3,7 +3,7 @@
 #include <opencv2/videoio.hpp>
 
 #include <opencv2/ovis.hpp>
-#include <opencv2/aruco_detector.hpp>
+#include <opencv2/objdetect/aruco_detector.hpp>
 
 #include <iostream>
 
diff -uraN opencv-4.9.0/opencv_contrib-4.9.0/modules/text/samples/text_recognition_cnn.cpp omv-opencv-4.9.0/opencv_contrib-4.9.0/modules/text/samples/text_recognition_cnn.cpp
--- opencv-4.9.0/opencv_contrib-4.9.0/modules/text/samples/text_recognition_cnn.cpp	2023-06-28 11:48:14.000000000 +0200
+++ omv-opencv-4.9.0/opencv_contrib-4.9.0/modules/text/samples/text_recognition_cnn.cpp	2023-07-16 13:15:37.309987059 +0200
@@ -38,7 +38,7 @@
         {
             Rect currrentBox = groups[indexes[i]];
             rectangle(src, currrentBox, Scalar( 0, 255, 255 ), 2, LINE_AA);
-            String label = format("%.2f", probs[indexes[i]]);
+           String label = cv::format("%.2f", probs[indexes[i]]);
             std::cout << "text box: " << currrentBox << " confidence: " << probs[indexes[i]] << "\n";
 
             int baseLine = 0;

diff -uraN opencv-4.9.0/opencv_contrib-4.9.0/modules/tracking/samples/benchmark.cpp omv-opencv-4.9.0/opencv_contrib-4.9.0/modules/tracking/samples/benchmark.cpp
--- opencv-4.9.0/opencv_contrib-4.9.0/modules/tracking/samples/benchmark.cpp	2023-06-28 11:48:14.000000000 +0200
+++ omv-opencv-4.9.0/opencv_contrib-4.9.0/modules/tracking/samples/benchmark.cpp	2023-07-16 14:09:27.302693692 +0200
@@ -159,7 +159,7 @@
     {
         Mat t, res;
         Mat(auc).convertTo(t, CV_64F); // integral does not support CV_32S input
-        integral(t.t(), res, CV_64F); // t is a column of values
+        cv::integral(t.t(), res, CV_64F); // t is a column of values
         return res.row(1) / (double)numTotal;
     }
 
