--- opencv-2.4.7/cmake/OpenCVFindLibsVideo.cmake.0000~	2013-11-23 10:13:34.248452176 +0100
+++ opencv-2.4.7/cmake/OpenCVFindLibsVideo.cmake	2013-11-23 10:14:51.595485205 +0100
@@ -127,6 +127,7 @@ ocv_clear_vars(HAVE_LIBV4L HAVE_CAMV4L H
 if(WITH_V4L)
   if(WITH_LIBV4L)
     CHECK_MODULE(libv4l1 HAVE_LIBV4L)
+    CHECK_MODULE(libv4l2 HAVE_LIBV4L)
   endif()
   CHECK_INCLUDE_FILE(linux/videodev.h HAVE_CAMV4L)
   CHECK_INCLUDE_FILE(linux/videodev2.h HAVE_CAMV4L2)
